plugins/azure/src/components/costmanagement/CostManagementToolbar.tsx

```diff
@@ -0,0 +1,29 @@
+import { Toolbar, ToolbarContent, ToolbarItem, ToolbarToggleGroup } from '@patternfly/react-core';
+import CostManagementToolbarItemTimeframe from './CostManagementToolbarItemTimeframe';
```

Comment:
Can you move this import to another group. I would like to have the first group with imports of external dependencies and a second one with imports from the project.